### PR TITLE
Fix all of the golint errors

### PR DESCRIPTION
**Pull request by bignastybryce**
Signed-off-by: Bryce Lynn <bryce.lynn@newcontext.com>
This fixes all of the "problems" that golint found in this repo.  They're almost all godoc comments.

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -159,6 +161,7 @@ func (v *ListCommand) execute() error {
 	return err
 }
 
+// GenerateAndStoreCommand contains the information needed to contruct a call to generate and store a cert
```
Comment (edwardbartholomew) [inline commands.go]: Nitpick typo (contruct vs construct)

File: commands.go [hunk 2]
```diff
@@ -72,6 +73,7 @@ func parseCommand() (Command, error) {
 	return v, nil
 }
 
+// ListCommand contains the information required to contruct a call to list certificates
```
Comment (edwardbartholomew) [inline commands.go]: Typo (contruct vs construct)

**Review by edwardbartholomew — CHANGES_REQUESTED** [review]
2 nitpick typos in comments

File: commands.go (hunk 2 above)
Comment (bignastybryce) [reply commands.go]: It's almost like I copied and pasted that...

File: commands.go (hunk 1 above)
Comment (bignastybryce) [reply commands.go]: Fixed.

**Review by edwardbartholomew — APPROVED** [review]
Looks great, thanks!

**Pull request merged**